zerver/lib/export.py

```diff
@@ -1962,6 +1961,8 @@ def export_realm_wrapper(
     print(f"Tarball written to {tarball_path}")
 
     if not upload:
+        shutil.rmtree(output_dir)
+        print(f"Successfully deleted the unpacked copy at {output_dir}")
```

Comment:
I don't think we need to print this line; it's a nice feature of the existing output that the last line we output is the path for the thing you wanted.

Also, I think we can just move the `shutil.rmtree(output_dir)` above to deduplicate code, perhaps in place of the "Finished exporting" line which is not useful now that we're deleting the thing at that path.